### ISSUE TITLE
Issue Installing v.3.12.3

### ISSUE THREAD

**Issue by FabioFleitas** [sass/node-sass#1805]
Currently having issues with v.3.12.3 that was recently released. See output below. When reverting to v.3.12.2, it works fine though.
```
-----> Node.js app detected
-----> Creating runtime environment
       
       NPM_CONFIG_LOGLEVEL=error
       NPM_CONFIG_PRODUCTION=true
       NODE_ENV=production
       NODE_MODULES_CACHE=true
-----> Installing binaries
       engines.node (package.json):  unspecified
       engines.npm (package.json):   unspecified (use default)
       
       Resolving node version (latest stable) via semver.io...
       Downloading and installing node 5.11.1...
       Using default npm version: 3.8.6
-----> Restoring cache
       Skipping cache restore (new runtime signature)
-----> Building dependencies
       Installing node modules (package.json)
       
       > node-sass@3.12.3 install /tmp/build_7e1004d4c1dbf13b8a4bde8acbbdab14/Tesorio-Tesorio-d7ba9bf16b9b5fac406eb02336bf803f35e4c33a/node_modules/node-sass
       > node scripts/install.js
       
       http node-sass install Downloading binary from https://github.com/sass/node-sass/releases/download/v3.12.3/linux-x64-47_binding.node
       [?25l
       throw new error.MissingTemplateValue(cloned, values)
       ^
       
       Error: Missing template value "progressbar"
       at cloneAndObjectify (/tmp/build_7e1004d4c1dbf13b8a4bde8acbbdab14/Tesorio-Tesorio-d7ba9bf16b9b5fac406eb02336bf803f35e4c33a/node_modules/gauge/render-template.js:67:17)
       at Array.map (native)
       at prepareItems (/tmp/build_7e1004d4c1dbf13b8a4bde8acbbdab14/Tesorio-Tesorio-d7ba9bf16b9b5fac406eb02336bf803f35e4c33a/node_modules/gauge/render-template.js:83:25)
       at module.exports (/tmp/build_7e1004d4c1dbf13b8a4bde8acbbdab14/Tesorio-Tesorio-d7ba9bf16b9b5fac406eb02336bf803f35e4c33a/node_modules/gauge/render-template.js:16:15)
       at Object.Plumbing.show (/tmp/build_7e1004d4c1dbf13b8a4bde8acbbdab14/Tesorio-Tesorio-d7ba9bf16b9b5fac406eb02336bf803f35e4c33a/node_modules/gauge/plumbing.js:45:10)
       at Object.Gauge._doRedraw (/tmp/build_7e1004d4c1dbf13b8a4bde8acbbdab14/Tesorio-Tesorio-d7ba9bf16b9b5fac406eb02336bf803f35e4c33a/node_modules/gauge/index.js:219:40)
       at Timeout.<anonymous> (/tmp/build_7e1004d4c1dbf13b8a4bde8acbbdab14/Tesorio-Tesorio-d7ba9bf16b9b5fac406eb02336bf803f35e4c33a/node_modules/gauge/index.js:15:19)
       at Timeout.wrapper (timers.js:408:11)
       at Timer.unrefdHandle (timers.js:445:14)
```

**Comment by njbarrett** [comment]
Seeing this also

**Comment by xzyfer** [comment]
Weird, can you please try v3.12.4

**Comment by aaronpearce** [comment]
Seeing this now too. Breaking on 3.12.4 as well for me.

**Comment by njbarrett** [comment]
3.12.4 seems to have fixed it for me

**Comment by xzyfer** [comment]
This should be fixed in 3.12.4. I'm prepared to rollback to 3.12.2 but I'd rather avoid it. 

@aaronpearce are you also using heroku? Can you please confirm you're using a fresh 3.12.4 install.

**Comment by aaronpearce** [comment]
Using capistrano to deply on EC2, will attempt a fresh install now.

**Comment by xzyfer** [comment]
Thank you. If you could also gist the npm installation log that'd be great.

**Comment by narthollis** [comment]
I am seeing this issue with 3.12.4 on our CI build server currently:

Excerpt from npm-debug.log
```
122156 verbose stack Error: node-sass@3.12.4 install: `node scripts/install.js`
122156 verbose stack Exit status 1
122156 verbose stack     at EventEmitter.<anonymous> (C:\Program Files\nodejs\node_modules\npm\lib\utils\lifecycle.js:242:16)
122156 verbose stack     at emitTwo (events.js:106:13)
122156 verbose stack     at EventEmitter.emit (events.js:191:7)
122156 verbose stack     at ChildProcess.<anonymous> (C:\Program Files\nodejs\node_modules\npm\lib\utils\spawn.js:40:14)
122156 verbose stack     at emitTwo (events.js:106:13)
122156 verbose stack     at ChildProcess.emit (events.js:191:7)
122156 verbose stack     at maybeClose (internal/child_process.js:852:16)
122156 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:215:5)
122157 verbose pkgid node-sass@3.12.4
122158 verbose cwd E:\agent-1\_work\33\s
122159 error Windows_NT 6.3.9600
122160 error argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install"
122161 error node v6.3.1
122162 error npm  v3.10.3
122163 error code ELIFECYCLE
122164 error node-sass@3.12.4 install: `node scripts/install.js`
122164 error Exit status 1
122165 error Failed at the node-sass@3.12.4 install script 'node scripts/install.js'.
122165 error Make sure you have the latest version of node.js and npm installed.
122165 error If you do, this is most likely a problem with the node-sass package,
122165 error not with npm itself.
122165 error Tell the author that this fails on your system:
122165 error     node scripts/install.js
122165 error You can get information on how to open an issue for this project with:
122165 error     npm bugs node-sass
122165 error Or if that isn't available, you can get their info via:
122165 error     npm owner ls node-sass
122165 error There is likely additional logging output above.
122166 verbose exit [ 1, true ]
```

**Comment by xzyfer** [comment]
@narthollis is there an error log also? I don't see the actual error in that excerpt.

**Comment by aaronpearce** [comment]
From a build run on CircleCI

`Error: Missing template value "progressbar"` from `node_modules/gauge/render-template.js:67:17`

**Comment by njbarrett** [comment]
Sorry, 3.12.4 didnt fix mine actually. Here's the error.

```
build	15-Nov-2016 11:21:59	> node-sass@3.12.4 install /var/atlassian/application-data/bamboo/xml-data/build-dir/FBC-FBCB-JOB1/node_modules/node-sass
build	15-Nov-2016 11:21:59	> node scripts/install.js
build	15-Nov-2016 11:21:59	
build	15-Nov-2016 11:21:59	http node-sass install Downloading binary from https://github.com/sass/node-sass/releases/download/v3.12.4/linux-x64-48_binding.node
build	15-Nov-2016 11:22:02	[?25l
error	15-Nov-2016 11:22:02	/var/atlassian/application-data/bamboo/xml-data/build-dir/FBC-FBCB-JOB1/node_modules/gauge/render-template.js:67
error	15-Nov-2016 11:22:02	          throw new error.MissingTemplateValue(cloned, values)
error	15-Nov-2016 11:22:02	          ^
error	15-Nov-2016 11:22:02	
error	15-Nov-2016 11:22:02	Error: Missing template value "progressbar"
error	15-Nov-2016 11:22:02	    at cloneAndObjectify (/var/atlassian/application-data/bamboo/xml-data/build-dir/FBC-FBCB-JOB1/node_modules/gauge/render-template.js:67:17)
error	15-Nov-2016 11:22:02	    at Array.map (native)
error	15-Nov-2016 11:22:02	    at prepareItems (/var/atlassian/application-data/bamboo/xml-data/build-dir/FBC-FBCB-JOB1/node_modules/gauge/render-template.js:83:25)
error	15-Nov-2016 11:22:02	    at module.exports (/var/atlassian/application-data/bamboo/xml-data/build-dir/FBC-FBCB-JOB1/node_modules/gauge/render-template.js:16:15)
error	15-Nov-2016 11:22:02	    at Object.Plumbing.show (/var/atlassian/application-data/bamboo/xml-data/build-dir/FBC-FBCB-JOB1/node_modules/gauge/plumbing.js:45:10)
error	15-Nov-2016 11:22:02	    at Object.Gauge._doRedraw (/var/atlassian/application-data/bamboo/xml-data/build-dir/FBC-FBCB-JOB1/node_modules/gauge/index.js:219:40)
error	15-Nov-2016 11:22:02	    at Timeout.<anonymous> (/var/atlassian/application-data/bamboo/xml-data/build-dir/FBC-FBCB-JOB1/node_modules/gauge/index.js:15:19)
error	15-Nov-2016 11:22:02	    at Timeout.wrapper (timers.js:425:11)
error	15-Nov-2016 11:22:02	    at Timer.unrefdHandle (timers.js:462:14)
```

**Comment by xzyfer** [comment]
Ok, I've released 3.12.5 which reverts the recent logging changes I suspect are the cause.

**Comment by xzyfer** [comment]
Please let me know if 3.12.5 has resolved your issues. If so I'll dig into what the root cause was when I have some time

**Comment by chriswait** [comment]
@xzyfer Thanks for the update! Are these reverted logging changes the same ones that fixed #1801 in PR #1803? (If so I'll probably hold off on updating, as I'll have CI build failures either way).

**Comment by xzyfer** [comment]
@chriswait yes. I suspect I'll revert the PR that introduced npmlog for logging also. It's caused a lot of trouble. 

Is someone able to confirm that this issue is resolved?

**Comment by narthollis** [comment]
Apologies, I missed that the error wasn't in the debug log. Error is as follows:

```
2016-11-15T03:03:47.5723458Z > node-sass@3.12.4 install E:\agent-1\_work\33\s\node_modules\node-sass
2016-11-15T03:03:47.5723458Z > node scripts/install.js
2016-11-15T03:03:48.7598348Z http node-sass install Downloading binary from https://github.com/sass/node-sass/releases/download/v3.12.4/win32-x64-48_binding.node
2016-11-15T03:03:51.4787748Z [?25l
2016-11-15T03:03:51.4787748Z [K[?25hE:\agent-1\_work\33\s\node_modules\gauge\render-template.js:67
2016-11-15T03:03:51.4787748Z           throw new error.MissingTemplateValue(cloned, values)
2016-11-15T03:03:51.4787748Z           ^
2016-11-15T03:03:51.4787748Z Error: Missing template value "progressbar"
2016-11-15T03:03:51.4787748Z     at cloneAndObjectify (E:\agent-1\_work\33\s\node_modules\gauge\render-template.js:67:17)
2016-11-15T03:03:51.4787748Z     at Array.map (native)
2016-11-15T03:03:51.4787748Z     at prepareItems (E:\agent-1\_work\33\s\node_modules\gauge\render-template.js:83:25)
2016-11-15T03:03:51.4787748Z     at module.exports (E:\agent-1\_work\33\s\node_modules\gauge\render-template.js:16:15)
2016-11-15T03:03:51.4787748Z     at Object.Plumbing.show (E:\agent-1\_work\33\s\node_modules\gauge\plumbing.js:45:10)
2016-11-15T03:03:51.4787748Z     at Object.Gauge._doRedraw (E:\agent-1\_work\33\s\node_modules\gauge\index.js:219:40)
2016-11-15T03:03:51.4787748Z     at Timeout.<anonymous> (E:\agent-1\_work\33\s\node_modules\gauge\index.js:15:19)
2016-11-15T03:03:51.4787748Z     at Timeout.wrapper (timers.js:421:11)
2016-11-15T03:03:51.4787748Z     at Timer.unrefdHandle (timers.js:458:14)
```

**Comment by xzyfer** [comment]
@mike-es thanks

@narthollis @njbarrett @aaronpearce can you confirm whether 3.12.5 fixed the issue for you?

**Comment by aaronpearce** [comment]
Attempting a build with it now.

**Comment by narthollis** [comment]
Yeah, just watched a build trundle through that installed 3.12.5 haply - Thanks.

```
2016-11-15T04:26:23.7416738Z > node-sass@3.12.5 install E:\agent-1\_work\33\s\node_modules\node-sass
2016-11-15T04:26:23.7416738Z > node scripts/install.js
2016-11-15T04:26:24.8667797Z info node-sass install Cached binary found at C:\Users\al.tfs\AppData\Roaming\npm-cache\node-sass\3.12.5\win32-x64-48_binding.node
2016-11-15T04:26:31.4136283Z > node-sass@3.12.5 postinstall E:\agent-1\_work\33\s\node_modules\node-sass
2016-11-15T04:26:31.4136283Z > node scripts/build.js
2016-11-15T04:26:31.5230346Z info node-sass build Binary found at E:\agent-1\_work\33\s\node_modules\node-sass\vendor\win32-x64-48\binding.node
2016-11-15T04:26:31.5230346Z info node-sass build Testing binary
2016-11-15T04:26:31.5699040Z info node-sass build Binary is fine
``

**Comment by FabioFleitas** [comment]
@xzyfer Can confirm that 3.12.5 works for me as well

**Comment by xzyfer** [comment]
Thanks everyone! I'm going to take this as motivation to drop npmlog.

**Comment by bryannaegele** [comment]
@xzyfer 3.12.5 resolves the issue for me from #1707 

Thanks for the prompt response and push.
